frontend/src/global_styles/content/work_packages/new/_overridden_description_wiki.sass

```diff
@@ -9,3 +9,9 @@
     // as its resized automatically by content (height)
     textarea
       resize: none
+
+    // As we currently do not have a fixed header in the creation page, we need to cancel out the padding of the page content.
+    // Otherwise, the scrolled content will be visible above the toolbar
+    // see: https://community.openproject.org/projects/openproject/work_packages/56921/activity
+    .document-editor__toolbar
+      top: -10px
```

Comment:
I assume it is not possible to use `$top-space` that sets the padding-top

Reply:
Unfortunately not really, as $top-space is a local variable. We'd have either have to move this whole block to the other file to make use of it, or import the other file in here which would duplicate the code in the compiled css.

Reply:
Since we are planning to rewrite the whole page in rails anyway and harmonize the layout of the `show` and the `new` page, we will hopefully get rid of this whole overwrite file anyway.